<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STEP_visu_2_prjClass</class>
 <widget class="QMainWindow" name="STEP_visu_2_prjClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STEP_visu_2_prj</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="pteFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbBrowse">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbParse">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Parse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbExit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFilter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbList">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClear">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Extract points</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>PRODUCT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>MANIFOLD_SOLID_BREP</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="lstList">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pbZoomIn">
        <property name="text">
         <string>Zoom in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbZoomOut">
        <property name="text">
         <string>Zoom out</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="cls_GraphicsWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cls_GraphicsWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>cls_GraphicsWidget.h</header>
   <slots>
    <slot>ZoomIn()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="step_visu_2_prj.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClear</sender>
   <signal>clicked()</signal>
   <receiver>lstList</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBrowse</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>SelectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbParse</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>ParseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbList</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>ListByFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>ListProducts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>ListSolids()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>STEP_visu_2_prjClass</receiver>
   <slot>ExtractPoints()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbZoomIn</sender>
   <signal>clicked()</signal>
   <receiver>openGLWidget</receiver>
   <slot>ZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectFile()</slot>
  <slot>ParseFile()</slot>
  <slot>ListByFilter()</slot>
  <slot>ListSolids()</slot>
  <slot>ListProducts()</slot>
  <slot>ExtractPoints()</slot>
 </slots>
</ui>
